packages/synapse-react-client/src/components/SchemaDrivenAnnotationEditor/template/FieldDescriptionTable.tsx

```diff
@@ -0,0 +1,60 @@
+import { Collapse } from '@mui/material'
+import React from 'react'
+import { JSONSchema7, JSONSchema7Definition } from 'json-schema'
+
+export type FieldDescriptionTableProps = {
+  schema: JSONSchema7
+  description: React.ReactNode
+}
+
+function getTypeAsString(
+  schema: JSONSchema7 | JSONSchema7Definition | JSONSchema7Definition[],
+): string {
+  if (Array.isArray(schema) || typeof schema === 'boolean') {
+    return 'unknown'
+  }
+  const { type, enum: _enum } = schema
+
+  if (type === 'array' && typeof schema.items === 'object') {
+    return `List of ${getTypeAsString(schema.items)}`
+  }
+
+  if (_enum) {
+    return 'enumeration'
+  }
+
+  if (Array.isArray(type)) {
+    return type.reduce(
+      (typesCommaSeparated, nextType) => `${typesCommaSeparated}, ${nextType}`,
+      '',
+    )
+  }
+  return type || 'unknown'
+}
+
+export default function FieldDescriptionTable(
+  props: FieldDescriptionTableProps,
+) {
+  const { description, schema } = props
+
+  const typeString = getTypeAsString(schema)
+
+  return (
+    <Collapse className="field-description" in={true}>
+      <table className="FieldDescriptionTable">
+        <tbody>
+          {description && (
+            <tr>
+              <th>Description</th>
+              <td>{description}</td>
+            </tr>
+          )}
+          <tr>
+            <th>Type</th>
+            <td>{typeString}</td>
+          </tr>
+        </tbody>
```

Comment:
No longer render a row for `required` because
1. I don't think we were accurately determining this value, since `required` is captured in the parent object in JSONSchema
2. The form already tells you if it's required with the `*`